README.md

```diff
@@ -135,6 +163,38 @@ To make this project runnable via IntelliJ, do the following:
 - Make sure that your configuration in `server/src/main/resources/reference.conf` 
   is configured according to your needs
 
+## How to Run Tests
+
+### Test controls
+
+The project uses the `.sbtrc` [(link)](https://www.scala-sbt.org/1.x/docs/Best-Practices.html#.sbtrc) file to manage and control the execution of integration tests. The Integration tests are the ones with the tag `IntegrationTest.` Below are the commands configured in the `.sbtrc` file to facilitate different testing scenarios:
```

Comment:
Theses descriptions might not be true. Because the need to switch java version complicates things and might prevent release, for now we dropping support for Spark 2.4 that requires it.

Also the above mentioned issue of `++2...` not working.

Reply:
Try to address by improving README texts - see commit 08171bab17a6f10188717d2696961e4a395fd6fb.